api/services/ocean/spark/schemas/oceanSparkConfigTemplate.yaml

```diff
@@ -0,0 +1,24 @@
+type: object
+required:
+- createdAt
+- updatedAt
+- id
+- displayName
+- clusterId
+- config
+allOf:
+- $ref: "../../../../commons/schemas/createdAtUpdatedAt.yaml"
+- type: object
+  description: >
+    The description of a configuration template
+  properties:
+    id:
+      $ref: "oceanSparkConfigTemplateId.yaml"
+    displayName:
+      $ref: "oceanSparkConfigTemplateDisplayName.yaml"
+    clusterId:
+      $ref: "oceanSparkClusterId.yaml"
+    config:
+      $ref: "oceanSparkApplicationSpec.yaml"
+    deletedAt:
```

Comment:
When is this field present? Can we list a deleted config template?
(Same question with other deletedAt fields in Apps, Jobs, ..)

Reply:
I defer to @crezvoy :)

Reply:
I don't see a reason for this field to be present publicly and I hardly see any reason for it to be present internally either. I think it could be removed from the object definition altogether.

Reply:
Thanks! Removing it